app/controllers/registers_controller.rb

```diff
@@ -57,6 +58,7 @@ def recover_records(fields, params)
     sort_direction = params[:sort_direction] ||= 'asc'
 
     @register.records
+             .where(entry_type: 'user')
```

Comment:
Could keep the scope and replace this with `@register.records.user_records`

Reply:
We weren't using the system_records scope anywhere so thought it seemed unnecessary to have user_records and system_records scoped? Now records is the same as what the user_records scope was.